README.md

```diff
@@ -52,7 +52,7 @@ If your Research Cloud account is set up for the Semantics of Sustainability pro
 ```
 - Close the terminal, open a new one, and run:
 ```console
-cd /scratch/tempo-embeddings/
+cd ../../opt/tempo-embeddings/
```

Comment:
I think this should be in line with the (absolute) directory as specified in [research_cloud/semantics_of_sustainability.yml](https://github.com/Semantics-of-Sustainability/tempo-embeddings/pull/60/files#diff-4d8cdea4c223f79aafb7ea898d774a812bdb2cd8cbeb82d7f9bd683201e739b6).
Using that absolute path instead of a relative path would also be less error-prone because the reader does not need to make assumptions about the current directory.
```suggestion
cd /opt/tempo-embeddings/
```